index.html

```diff
@@ -9,31 +9,30 @@
     <!-- <script src="https://cdn.tailwindcss.com"></script> -->
     <link rel="stylesheet" href="tailwind.min.css" />
     <link rel="stylesheet" href="utilities.min.css" />
+    <link rel="stylesheet" href="style.css" />
   </head>
-  <body class="h-full w-full flex flex-col bg-[#ccffaa] text-dark">
+  <body class="h-full w-full flex flex-col bg-[#ccffaa] text-[#229922]">
     <h1 class="text-3xl font-thin text-center mt-4">Slabsprak</h1>
-    <div class="h-full w-full flex flex-col justify-around items-center">
+    <div class="h-full flex flex-col justify-around items-center">
       <input
         id="input"
         autofocus="true"
         placeholder="get your snacks"
-        class="text-center outline-none bg-[#ccffaa] rounded-md p-2"
```

Comment:
I keep changing this to {outline: none} rather than zero because the highlight continues to show on every browser I try. Not sure if your experience is different here but it might need a belt and braces approach just to be sure